ci/deploy.sh

```diff
@@ -85,6 +85,21 @@ build_pyfory() {
     else
       $PYTHON_CMD setup.py bdist_wheel --dist-dir="$ROOT/dist"
     fi
+  elif [[ "$OSTYPE" == "msys" || "$OSTYPE" == "win32" ]]; then
+
+    if [ -n "$GITHUB_REF_NAME" ]; then
+      # Strip leading 'v' if present
+      version="${GITHUB_REF_NAME#v}"
```

Comment:
This may not generate right verson, the release.py have special handle for version, which change `0.12.1-rc1` to `0.12.1rc1`:
```python
def _update_python_version(lines, v: str):
    for index, line in enumerate(lines):
        if "__version__ = " in line:
            v = v.replace("-alpha", "a")
            v = v.replace("-beta", "b")
            v = v.replace("-rc", "rc")
            lines[index] = f'__version__ = "{v}"\n'
            break
```

Reply:
I can't call python in bash, so I'll implement this function in it.

Reply:
`version="${version//-alpha/a}" && version="${version//-beta/b}" && version="${version//-rc/rc}"` this should work